samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/NestedSamples/SafeArea_ModalPage.xaml

```diff
@@ -9,8 +9,8 @@
 	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
 	<Grid utu:SafeArea.Insets="VisibleBounds"
-		  utu:SafeArea.SafeAreaOverride="0,0,0,30"
```

Comment:
Not working as it fails at runtime ?

Reply:
Fails to build actually

Reply:
What's the error ?

Reply:
I'm assuming it's something to do with that property being internal. Even though Internals should be visible to Uno.Toolkit.Samples and Uno.Toolkit.Samples.UWP

Reply:
The internal dp is visible to our sample project assembly. You can call this in the code-behind and it would build:
```cs
Uno.Toolkit.UI.SafeArea.SafeAreaOverrideProperty.ToString();
```
The problem however, is that the page is not "loaded" by our assembly, but:
```cs
global::System.Uri resourceLocator = new global::System.Uri("ms-appx:///Content/NestedSamples/SafeArea_ModalPage.xaml");
global::Windows.UI.Xaml.Application.LoadComponent(this, resourceLocator, global::Windows.UI.Xaml.Controls.Primitives.ComponentResourceLocation.Application);
```

Reply:
none of them works:
```cs
[assembly: InternalsVisibleTo("UniversalWindowsPlatform")]
[assembly: InternalsVisibleTo("Windows.Foundation.UniversalApiContract")]
[assembly: InternalsVisibleTo("Windows.UI.Xaml")]
[assembly: InternalsVisibleTo("Windows.UI.Xaml.Controls")]
[assembly: InternalsVisibleTo("Windows.UI.Xaml.Markup")]

[assembly: InternalsVisibleTo("Microsoft.UI.Xaml")]
[assembly: InternalsVisibleTo("Microsoft.UI.Xaml.Controls")]
[assembly: InternalsVisibleTo("Microsoft.UI.Xaml.Markup")]

[assembly: InternalsVisibleTo("PresentationBuildTasks")]
[assembly: InternalsVisibleTo("Microsoft.Build.Tasks.Windows")]
[assembly: InternalsVisibleTo("Microsoft.Windows.UI.Xaml.Build.Tasks")]
[assembly: InternalsVisibleTo("Microsoft.Windows.UI.Xaml.Common")]

[assembly: InternalsVisibleTo("Microsoft.NETCore.UniversalWindowsPlatform")]
```